### DISCUSSION TITLE
What's our name? What's our URL?

### DISCUSSION THREAD

**Discussion by reece** [HGVSnomenclature/hgvs-nomenclature#112] [HGVS nomenclature proposals]
Two related questions:

- **What's our name?** This project has had lots of names. From the earliest draft and discussions, I *thought* that there was agreement to call this project HGVS Nomenclature and that we should deprecate historical names (mutnoment, varnomen, etc). We changed the github org name, the repo name, and have been using HGVS Nomenclature for a year. 

- **What's our URL?** Related to the above, with every change of name and host organization, we have changed the name. The URL has been unstable. Instead, we should use a name that can stay with the project regardless of affiliation.

**Comment by reece** [comment]
The genesis of this discussion is @jtdendunnen's proposal to keep varnomen.hgvs.org as the primary name, which promulgates the varnomen name. It's also exactly the opposite of the proposal to forward from varnomen in https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/17 for the last year.

**Reply by ahwagner** [reply]
HGVS Nomenclature is the clearer name for the product. I vote we use that.

**Comment by ifokkema** [comment]
I personally have no strong preference; the domain name looks more professional and clear, especially for new users, but the original site is still remembered by many and likely bookmarked and linked to from publications and websites. If the technical setup is easier on the domain name, I'd vote to move there.

The only request I have is that the original _varnomen.hgvs.org_ URLs will automatically redirect to the new ones. This will keep all bookmarks and old links functional. The redirect could use a `HTTP 301` to indicate to systems the old URL should be retired.

**Reply by reece** [reply]
I can think of these options for forwarding:

1. banner at the top -- I can do this in 2 minutes total
2. manual per-page forwarding (i.e., user must click a link)
3. automatic per-page forwarding (i.e., page is forwarded automatically)
4. DNS CNAME update

I prefer 1 or 2. Manual forwarding gives people the opportunity to read the old pages, whereas options 3 and 4 would make the old pages inaccessible.

**Reply by ahwagner** [reply]
I agree that hgvs-nomenclature.org reinforces the name of the specification, and therefore improves clarity and consistency. I vote we move there.

I think that @reece's proposal to redirect with a banner is a good solution (and we have already seen it implemented); but agree with @ifokkema that we will want to eventually move to automated redirects after a deprecation period (I usually use a 6 month window for these types of things).

My preference would be to do the following: 
1) stand up a clone of `varnomen.hgvs.org` at `archive.hgvs-nomenclature.org`
2) automated traffic redirect from `varnomen.hgvs.org` to `hgvs-nomenclature.org`
3) add a banner to the top of `hgvs-nomenclature.org` that says something like "We have moved! Looking for the old site? Visit `archive.hgvs-nomenclature.org`"

In this way, users are redirected to the newest version of the spec (at `hgvs-nomenclature.org`) but are provided the means to visit the old pages (at `archive.hgvs-nomenclature.org`) for as long as HVNC wants to keep them accessible.

**Reply by ifokkema** [reply]
@ahwagner I like that suggestion, too. It's the best of both worlds; people always have the newer specs and won't have to see/read the banner to get there, but when "surprised" by the new website, they will stop and read and find a way to get to the old site if they really want to.

**Comment by reece** [comment]
Decided at HVNC meeting on 12/4: The name is HGVS Nomenclature and the URL is hgvs-nomenclature.org.

Separately, develop a plan for archiving current pages and arranging forwarding. @ahwagner 's suggestion on https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/112#discussioncomment-7735515, or something like it, was favored.
